src/main/resources/db/migration/V4__translation_offer.sql

```diff
@@ -9,6 +9,7 @@ create table translation_offer
     region        text,
     description   text,
     modified_date timestamp not null,
+    status        text,
```

Comment:
zamiast modyfikować skrypty lepiej zdecydowanie dodać nowy w kolejności który doda dodatkowe pole do pól przez ALTER TABLE {name} ADD {column name} {type} DEFAULT {default value} wtedy nie będzie konfliktów w bazie

Reply:
Dzięki - zmienię, pomyślałam, że póki nie jesteśmy zreleasowani to da nam to większy porządek w migracjach.